vcd/resource_vcd_nsxt_alb_pool.go

```diff
@@ -442,6 +452,9 @@ func setNsxtAlbPoolData(d *schema.ResourceData, albPool *types.NsxtAlbPool) erro
 	dSet(d, "default_port", albPool.DefaultPort)
 	dSet(d, "graceful_timeout_period", albPool.GracefulTimeoutPeriod)
 	dSet(d, "passive_monitoring_enabled", albPool.PassiveMonitoringEnabled)
+	if albPool.SslEnabled != nil {
+		dSet(d, "ssl_enabled", *albPool.SslEnabled)
```

Comment:
The new Optional + Computed approach looks great! Just one question about the above line: what happens when we set it behind the scenes? How does the next plan look like?

Thanks!

Reply:
The property is `Optional`+`Computed`. Thus its value is reported without comparison.
If we remove `Computed`, the plan fails.